README.md

```diff
@@ -204,8 +205,8 @@ ufw_outgoing_traffic:
   - 443
   - 853
 ```
-
-Open `ufw` ports, allowing outgoing traffic.
+`ufw_enable: false` gives you the opportunity to manage your own firewall and firewall rules.
```

Comment:
I think this needs a rewording, to reduce any confusion we should probably describe what the default does.
"`ufw_enable: true` enables `ufw` and related firewall rules. If you want to manage your own firewall and firewall rules, this should be `false`" or something similar.

Reply:
I've tried something, I'll let you review it and let me know if this is more clear now. English not being my mother tongue, my sentence can undoubtedly be improved, especially when this comes to documentation :)